src/app/cli/src/explore/web_ui_server.rs

```diff
@@ -150,10 +148,17 @@ impl WebUIServer {
             .await
             .expect("Token not retrieved");
 
+        let graphql_router = OpenApiRouter::new()
+            .route("/graphql", axum::routing::post(graphql_handler))
+            .layer(graphql_http::middleware::GraphqlTracingLayer::new(
+                kamu_adapter_graphql::schema(),
+                kamu_adapter_graphql::schema_quiet(),
```

Comment:
minor: Would it be better to change return type to `fn schema() -> SchemaBuilder` and let tracing layer attach necessary extension?

Reply:
I wanted to do it this way at first, but `async_graphql::SchemaBuilder<Q, M, S>` does not implement `Clone`, and is also consumed by `build()`.

Considering that the schemas are cheap to clone, I suggest leaving as is